src/view/pages/Certification/CertBodyInfo.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useQuery } from 'react-query';
+import { Typography, Grid } from '@material-ui/core';
+import { LoadingWithMinDisplay, SpinnerWithLabel } from 'view/components';
+import { fetchOrganizationById, CertResData } from 'services/api';
+import { InfoItem } from 'view/components/InfoItem';
+
+interface Certificate {
+  certificate: CertResData;
+}
+
+export function CertBodyInfo({ certificate }: Certificate) {
+  const { certifying_body_id, certifying_body } = certificate;
+  const { isLoading, error, data } = useQuery('fetchOrganizationById', () =>
+    fetchOrganizationById(certifying_body_id),
+  );
+
+  return (
+    <LoadingWithMinDisplay
+      isLoading={isLoading}
+      loadingIndicator={<SpinnerWithLabel label="Loading factory info..." />}
```

Comment:
```suggestion
      loadingIndicator={<CircularProgress />}
```

I don't have this codified anywhere, but I have been trying to use the `<SpinnerWithLabel />` component mostly for full-page loading, or more generally for a top-level component. 

In this scenario where we are just loading a single component in a larger top-level component, I think it makes more sense to just throw a regular `<CircularProgress />` component into the `loadingIndicator` since the additional context provided by the label isn't really needed IMO.